app/grandchallenge/archives/tasks.py

```diff
@@ -39,34 +35,3 @@ def add_images_to_archive(*, upload_session_pk, archive_pk, interface_pk=None):
                 continue
             item = ArchiveItem.objects.create(archive=archive)
             item.values.set([civ])
-
-
-@acks_late_micro_short_task
-def add_images_to_archive_item(
```

Comment:
You cannot just remove celery tasks as there may be existing ones on the queue. You need to leave them in and handle the migration, deploy, then remove the task, then deploy again.

Reply:
If you don't do that and there is an old task then celery panics about not having the task, makes an exception, puts the message back on the queue, flooding sentry.

Reply:
Ah, I hadn't considered two-step this to clear the queues.  Good tip.